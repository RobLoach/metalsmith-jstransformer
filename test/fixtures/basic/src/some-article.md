---
title: Title
---
This is the contents.
