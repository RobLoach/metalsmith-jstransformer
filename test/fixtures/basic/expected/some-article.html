<p>This is the contents.</p>
